perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/ListBox.java

```diff
@@ -0,0 +1,68 @@
+package cz.metacentrum.perun.wui.registrar.widgets.items;
+
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.user.client.ui.HasVerticalAlignment;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.user.client.ui.VerticalPanel;
+import org.gwtbootstrap3.client.ui.Input;
+import cz.metacentrum.perun.wui.widgets.PerunButton;
+import cz.metacentrum.perun.wui.model.beans.ApplicationFormItemData;
+import cz.metacentrum.perun.wui.registrar.widgets.PerunForm;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents ListBox form item.
+ *
+ * @author Jakub Hejda <Jakub.Hejda@cesnet.cz>
+ */
+public class ListBox extends WidgetBox {
+
+	List<Input> inputList = new ArrayList<>();
+
+	public ListBox(PerunForm form, ApplicationFormItemData item, String lang) {
+		super(form, item, lang);
+	}
+
+	@Override
+	public String getValue() {
+		String value = "";
+		for (Input input : inputList) {
```

Comment:
You probably want to use our `ExtendedTextBox` instead of plain `Input` to take advantage of copy/past, regex validation etc.